### PR TITLE
Flutter run machine

### PR DESCRIPTION
**Pull request by devoncarew**
- update IntelliJ to use `flutter run --machine` (instead of flutter daemon `app.startApp`) when starting an app (fix https://github.com/flutter/flutter-intellij/issues/401). We still use the daemon protocol, but use it in the one-shot manner provided by flutter run --machine

@pq, @danrubel

### CODE REVIEW

File: src/io/flutter/run/daemon/DaemonJsonInputFilterProvider.java [hunk 1]
```diff
@@ -31,9 +31,14 @@
     public List<Pair<String, ConsoleViewContentType>> applyFilter(String text, ConsoleViewContentType contentType) {
       if (!VERBOSE) {
```
Comment (danrubel) [inline src/io/flutter/run/daemon/DaemonJsonInputFilterProvider.java]: Is the VERBOSE final ever set true? Should it be removed and this code simplified?

Comment (devoncarew) [inline src/io/flutter/run/daemon/DaemonJsonInputFilterProvider.java]: It's just for use during development - when you're working on the launch code itself.

**Comment by devoncarew** [comment]
Re: the version check, we did bump to needing `0.0.3` (as part of the --machine change).

**Comment by devoncarew** [comment]
Thanks for the reviews!

**Pull request merged**